Documentation/platforms/arm/rp2040/boards/seeed-xiao-rp2040/README.txt

```diff
@@ -0,0 +1,100 @@
+README
+======
```

Comment:
move to Documentation folder

Reply:
Hi @xiaoxiang781216 , 
This README.txt file is already on Documentation folder (Documentation/platforms/arm/rp2040/boards/seeed-xiao-rp2040/README.txt), so I`m not sure what I need to do here, could you please let me know ?
Thanks

Reply:
Sorry, I mean change to .rst format, but I saw many README is still in text format. So, please ignore my comment.